templates/bundles/DemosPlanStatementBundle/DemosPlanAssessment/view_statement.html.twig

```diff
@@ -1,5 +1,19 @@
 {% extends '@DemosPlanCore/DemosPlanCore/base.html.twig' %}
 
+{# if there are statement votes ("Mitzeichner"), then the option "Schlussmitteilung versenden" plus corresponding hint will be shown #}
+{% set templateVars.sendFinalEmail = true %}
+{% set templateVars.finalEmailOnlyToVoters = false %}
+{% if templateVars.table.statement.feedback == 'snailmail' %}
+  {% set templateVars.table.statement.feedback = "via.post"|trans %}
+  {% if templateVars.table.statement.votes is empty %}
+    {% set templateVars.sendFinalEmail = false %}
+  {% else %}
+    {% set templateVars.finalEmailOnlyToVoters = true %}
+  {% endif %}
+{% elseif templateVars.table.statement.feedback == 'email' %}
+    {% set templateVars.table.statement.feedback = "via.email"|trans %}
+{% endif %}
```

Comment:
Please don't use `templateVars` for the vars not coming from the backend anymore.

Reply:
Was necessary to avoid refactoring of this twig file and its `include`s.

Reply:
there are not so much occurrences. And I don't wan't to reduce it technical dept in the controller but produce it here

Reply:
If it can be simplified I'm totally fine with that. @spiess-demos refactored the file, please review.

Reply:
If it can be simplified I'm totally fine with that. @spiess-demos refactored the file, please review.